src/App.tsx

```diff
@@ -16,35 +22,96 @@
 ];
 
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState<SortType>(SortType.DEFAULT);
+  const [isReversed, setIsReversed] = useState<boolean>(false);
+
+  const handleReverse = () => {
+    setIsReversed(!isReversed);
+  };
+
+  const handleReset = () => {
+    setSortField(SortType.DEFAULT);
+    setIsReversed(false);
+  };
+
+  const getSortedGoods = () => {
```

Comment:
[CODE STYLE] - The function name 'getSortedGoods' does not start with a verb. Consider renaming it to 'sortGoods' or 'getSortedGoodsList'.